src/Orneholm.CognitiveWorkbench.Web/Views/Shared/_RecognizeTextOperationResult.cshtml

```diff
@@ -0,0 +1,84 @@
+@using Orneholm.CognitiveWorkbench.Web.Extensions
+@model Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models.TextOperationResult
+
+<div class="mb-5">
+    <h4>Metadata</h4>
+
+    <table class="table table-striped table-hover">
+        <thead class="thead-dark">
+            <tr>
+                <th>Key</th>
+                <th>Value</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <th>Status</th>
+                <td>@Model.Status</td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+
+
+<div class="my-5">
+    <h4>Lines</h4>
+
+    <table class="table table-striped table-hover">
+        <thead class="thead-dark">
+            <tr>
+                <th>Rectangle</th>
+                <th>Words</th>
+            </tr>
+        </thead>
+        <tbody class="cwb-image-boxes-table" data-cwb-for=".cwb-image-boxes-ocr-lines">
```

Comment:
Change to `.cwb-image-boxes-recognizetext-lines`

Reply:
Changing it in an incoming update

In that case, it will be like the following as I did not edit the css

![image](https://user-images.githubusercontent.com/6305666/76312474-704df480-62d3-11ea-90f3-3deb7417657b.png)